bin/templates/scripts/cordova/run.bat

```diff
@@ -15,11 +15,5 @@
 :: specific language governing permissions and limitations
 :: under the License
 @ECHO OFF
-SET script_path="%~dp0run"
-IF EXIST %script_path% (
-        node %script_path% %*
-) ELSE (
-    ECHO.
-    ECHO ERROR: Could not find 'run' script in 'cordova' folder, aborting...>&2
-    EXIT /B 1
-)
+ECHO ERROR: Running 'ios' platform on Windows is not supported>&2
+EXIT /B 0
```

Comment:
Why are you exiting with a success status?  The previous version exited with status code 1.  

Probably should also change build.bat to return status code 1.